src/data/es/index.ts

```diff
@@ -105,11 +110,6 @@ export const instanceData = {
       applet: 'Load Applet from %provider%',
       twingle: 'Load Donation Form'
     },
-    notifications: {
-      notifications: "Notificaciones",
-      pleaseLogInLink: "Por favor, inicia sesión",
-      pleaseLogInText: "para ver tus notificaciones."
```

Comment:
For example: This three translations will be lost?

Reply:
Ah, ok. That is intended. The guard component now has more general strings, so we don't have to add new strings for every function.
I will solve the conflicts and merge.